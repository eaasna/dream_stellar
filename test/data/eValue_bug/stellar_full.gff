2	Stellar	eps-matches	189	383	95.4081	+	.	1;seq2Range=664,859;eValue=1.66866e-82;cigar=191M1I4M;mutations=19C,71G,79C,93C,111C,119T,187A,192A,195G
2	Stellar	eps-matches	68	253	95.2127	+	.	1;seq2Range=446,632;eValue=2.27431e-78;cigar=2M1I172M1I1M1D10M;mutations=3A,46C,63G,96C,105T,135G,168C,176C
1	Stellar	eps-matches	177	356	95.0276	+	.	1;seq2Range=1807,1986;eValue=1.1451e-74;cigar=92M1D1M1I86M;mutations=54C,70A,84T,94G,109A,139C,150G,163C
1	Stellar	eps-matches	126	302	95.4802	+	.	1;seq2Range=204,378;eValue=3.76404e-74;cigar=2M2D173M;mutations=19A,24T,96G,159C,166A,170A
1	Stellar	eps-matches	28	203	95.4545	+	.	1;seq2Range=1201,1376;eValue=1.23727e-73;cigar=176M;mutations=40T,54G,59T,105C,118A,135C,167T,170A
3	Stellar	eps-matches	1	173	95.4022	+	.	1;seq2Range=686,859;eValue=4.76748e-73;cigar=169M1I4M;mutations=49G,57C,71C,89C,97T,165A,170A,173G
1	Stellar	eps-matches	276	426	95.3642	+	.	1;seq2Range=2111,2261;eValue=2.89931e-62;cigar=151M;mutations=4A,9A,19C,40C,48C,97C,134G
2	Stellar	eps-matches	20	147	95.3125	+	.	1;seq2Range=1056,1183;eValue=5.17824e-52;cigar=128M;mutations=12A,22A,33T,39T,52A,63G
1	Stellar	eps-matches	469	557	95.5056	+	.	1;seq2Range=60,148;eValue=7.13259e-35;cigar=89M;mutations=15G,58C,60A,87A
2	Stellar	eps-matches	148	201	96.3636	+	.	1;seq2Range=912,966;eValue=1.73628e-20;cigar=52M1I2M;mutations=26A,53T
1	Stellar	eps-matches	75	125	96.0784	+	.	1;seq2Range=1643,1693;eValue=2.46139e-18;cigar=51M;mutations=8G,24T
3	Stellar	eps-matches	1	31	96.7741	+	.	1;seq2Range=590,620;eValue=5.35996e-10;cigar=31M;mutations=24C
2	Stellar	eps-matches	1	23	95.6521	+	.	1;seq2Range=152,173;eValue=1.68702e-05;cigar=20M1D2M;mutations=
1	Stellar	eps-matches	2	11	100	+	.	1;seq2Range=1921,1930;eValue=3.0168;cigar=10M;mutations=
2	Stellar	eps-matches	422	431	100	+	.	1;seq2Range=539,548;eValue=2.48442;cigar=10M;mutations=
3	Stellar	eps-matches	34	43	100	+	.	1;seq2Range=623,632;eValue=1.07584;cigar=10M;mutations=
1	Stellar	eps-matches	453	462	100	-	.	1;seq2Range=114,123;eValue=3.0168;cigar=10M;mutations=
2	Stellar	eps-matches	158	167	100	-	.	1;seq2Range=1105,1114;eValue=2.48442;cigar=10M;mutations=
