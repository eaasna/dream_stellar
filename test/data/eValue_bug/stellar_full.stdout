I/O options:
  database file   : ref.fasta
  query file      : query_e0.05.fasta
  alphabet        : dna5
  output file     : stellar_full.gff
  output format   : gff

User specified parameters:
  minimal match length             : 10
  maximal error rate (epsilon)     : 0.05 = (1/20)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 10
  s^min        : 10
  threshold    : 1
  distance cut : 10
  delta        : 16
  overlap      : 0

Loaded 1 query sequence.
Loaded 3 database sequences.

All matches matches resulting from your search have an E-value of: 
        3.0168 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  1
  2
  3

16

16

16

16

16

16

16

16

16

16

16

16

16

16

16

16
  1, complement
  2, complement
  3, complement

16

16

# Eps-matches     : 18
